canonical/canonical-design
# Wd 16710 shift current static pages to jinja templating

## Done
- Moved pages to jinja templating
- Updated Navigation
## QA
- Run the site locally using dotrun and see if you can now visit all the pages via navigation.
## Fixes
- [WD-16710](https://warthogs.atlassian.net/browse/WD-16710)


[WD-16710]: https://warthogs.atlassian.net/browse/WD-16710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
